frontend/src/app/modules/home/dashboard/dashboard.component.html

```diff
@@ -24,19 +28,37 @@ <h2 appTooltipFullName>{{dashboard?.name}}</h2>
         <app-no-content-placeholder title="Looks like here are no tiles"></app-no-content-placeholder>
     </div>
     <div *ngIf="tiles.length" class="tiles-container">
-        <ng-container *ngFor="let tile of tiles" [ngSwitch]="tile.type">
-            <app-top-active-issues-tile (isDeleting)="deleteTile($event)" *ngSwitchCase="tileTypes.TopActiveIssues"
-                [isShownEditTileMenu]="showTileMenu" [tile]="tile" [userProjects]="projects"
-                [style.flex-basis.px]="getTileSize(tile)" class="top-active-issue">
-            </app-top-active-issues-tile>
-            <app-issues-per-time-tile (isDeleting)="deleteTile($event)" *ngSwitchCase="tileTypes.IssuesPerTime"
-                [isShownEditTileMenu]="showTileMenu" [tile]="tile" [userProjects]="projects"
-                [style.flex-basis.px]="getTileSize(tile)" class="issues-per-time">
-            </app-issues-per-time-tile>
-            <app-issues-count-tile (isDeleting)="deleteTile($event)" *ngSwitchCase="tileTypes.IssuesCount"
-                [isShownEditTileMenu]="showTileMenu" [tile]="tile"
-                [userProjects]="projects" [style.flex-basis.px]="getTileSize(tile)" class="issues-count">
-            </app-issues-count-tile>
+        <ng-container *ngFor="let tile of tiles; let i = index" [ngSwitch]="tile.type">
+            <div
+                class="tile"
+                (onDragEnd)="dragOff()"
+                (onDragStart)="drag(tile)"
+                [style.flex-basis.px]="getTileSize(tile)"
+                [pDraggableDisabled]="!canDrag"
+                [pDraggable]="canDrag? 'dd' : null ">
+                <div
+                    (onDrop)="drop(i)"
+                    class="drop-line drop-line-left"
+                    *ngIf="showTileMenu && ((tiles.indexOf(draggableTile) !== i && tiles.indexOf(draggableTile) !== i-1) || !draggableTile)"
```

Comment:
It would be better to move it to a separate method

Reply:
Fixed